contrib/ci/check-dep-versions.py

```
@@ -216,6 +216,42 @@ def sync_xml_versions(xml_file: str, meson_versions: Dict[str, str]) -> List[str
+    will never be matched at runtime."""
+
+    # Variant names that are mapped to something else and thus must not
+    # appear in the XML.
+    FORBIDDEN_VARIANTS: Dict[str, str] = {
+        "amd64": "x86_64",
+        "arm": "armhf",
+        "arm64": "aarch64",
+    }
+
+    rc = 0
+    tree = etree.parse(xml_file)
```

Ruff (0.16.0)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)